anvio/__init__.py

```diff
@@ -396,14 +396,28 @@ def get_args(parser):
              'help': "Takes a value between 0 and 1, where 1 means that only functions that occur in all members of\
                       one of the compared groups will be included in the output. Default is %(default).1f."}
                 ),
+    'false-detection-rate': (
+            ['--false-detection-rate', '--FDR'],
+            {'metavar': 'PORTION',
+             'default': 0.05,
+             'type': float,
+             'help': "Takes a value between 0 and 1, to determine the false detection rate that will be used \
+                      for the Benjamini–Hochberg procedure. Default is %(default).1f."}
+                ),
+    'core-threshold': (
+            ['--core-threshold'],
+            {'metavar': 'PORTION',
```

Comment:
metavar is much more useful when it describes the data type. integer, float, name, file path, etc :)

Reply:
Also done :-)
https://github.com/merenlab/anvio/commit/9e2c33c7c1e8dc99a938b19711a1e686127d10ff